Src/Fido2.Development/StoredCredential.cs

```diff
@@ -9,7 +9,7 @@ public class StoredCredential
     /// <summary>
     /// The Credential ID of the public key credential source.
     /// </summary>
-    public byte[] Id { get; set; }
+    public required byte[] Id { get; set; }
```

Comment:
Very happy to see we're on .NET8, and are able to use a required property here.

Reply:
Can this also be made immutable with { get; init; }?

Reply:
@iamcarbon It probably could, but that opens the question if not more of these should be made requried/init-only etc so in an effort to merge this I won't add it.

Also - I need to remind myself that this type is only used in the Development namespace, so it's just used for demoes etc.d